src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Triggers/QueueTriggerAttributeBindingProvider.cs

```diff
@@ -45,14 +45,14 @@ internal class QueueTriggerAttributeBindingProvider : ITriggerBindingProvider
             _loggerFactory = loggerFactory;
         }
 
-        public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext context)
+        public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext context)
         {
             ParameterInfo parameter = context.Parameter;
             var queueTrigger = TypeUtility.GetResolvedAttribute<QueueTriggerAttribute>(context.Parameter);
 
             if (queueTrigger == null)
             {
-                return null;
+                return Task.FromResult<ITriggerBinding>(null);
```

Comment:
Fixed in latest: https://github.com/Azure/azure-webjobs-sdk/blob/3c3fc7dfa107bd65f1ea4e6d03580de66f404695/src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Triggers/QueueTriggerAttributeBindingProvider.cs#L82